src/vscode-bicep/src/visualizer/app/assets/icons/azure/index.ts

```diff
@@ -161,6 +219,12 @@ export async function importResourceIconInline(
       return (
         await import("./networking/10372-icon-service-Public-IP-Prefixes.svg")
       ).default;
+    case "microsoft.network/applicationsecuritygroups":
+      return (
+        await import(
+          "./security/10244-icon-service-Application-Security-Groups.svg"
```

Comment:
It seems this icon file is not added.

Reply:
Doh, it was in the wrong folder. Updated.